doc-site/docs/concepts/tokens.md

```diff
@@ -0,0 +1,104 @@
+
+### Built on EVM, and providing programmability through EVM
+
+EVM is the place where the most innovation is happening on privacy technologies. EVM based ledgers provide all the tools needed to build layers of privacy on top, and most importantly to make those private worlds interoperable with other worlds on the same ledger.
+
+Paladin is designed to unleash the innovation and talent of the EVM ecosystem, for enterprise use cases that have a fundamental requirement for privacy.
+
+Paladin not only uses EVM as the base ledger it runs against (it is compatible with any EVM), but it allows programmability through EVM.
+
+### The building blocks
+
+Constructing a tokenization use case has historically mean choosing between three very different approaches to maintaining privacy:
+
+1. Zero-knowledge proof (ZKP) based tokens
+    - These obey the purest principals of decentralization, where mathematics enforces the rules of the token. No individual party is required to be involved in any processing, and by the nature of that no party can prevent a valid transaction from executing.
+    - Cash tokens, including Central Bank Digital Currencies (CBDCs), are driving enterprise adoption of this form of token due to the simple mathematical nature of the tokens. Control points are often still required such as KYC/AML checks on the governance of parties authorized to trade, and activities that affect overall supply.
+    - The overhead of the cryptography is comparatively high with this set of technologies, so there is rapid evolution in the toolkits and approaches to reduce the overhead and increase efficiency.
+2. Issuer-backed tokens / Private Oracles / Notarized tokens
+    - These do not execute entirely in the open, but instead require some centralization of control - most commonly because the legal framework they operate in require it. The fact there is a single party means that anonymity and confidentiality between trading parties can be established by inclusion of the operator/issuer in each transaction, and trusting that operator to maintain the private date they are legally required to hold. That party controls what data is shared with whom.
+    - Regulated securities such as Bonds and Funds almost always fall into this category, where existing legal frameworks for the governance of ownership records mean that some set of legal entities must be involved in any transfer, and must have full details of that transfer.
+3. Private smart contracts
+    - The power of an smart contract, EVM or otherwise, is that every party involves knows the same logic is being executed against the same data, in the same order. Previous projects, including ones with significant adoption in the enterprise EVM community such as [Tessera](https://github.com/connsensys/tessera), have aimed to provide this same benefit within **privacy groups** sharing a common ledger.
```

Comment:
```suggestion
    - The power of an smart contract, EVM or otherwise, is that every party involves knows the same logic is being executed against the same data, in the same order. Previous projects, including ones with significant adoption in the enterprise EVM community such as [Tessera](https://github.com/Consensys/tessera), have aimed to provide this same benefit within **privacy groups** sharing a common ledger.
```